Packs/SailPointIdentityIQ/Integrations/SailPointIdentityIQ/SailPointIdentityIQ.py

```diff
@@ -366,16 +368,25 @@ def search_identities(client: Client, id: str, email: str, risk: int, active: bo
     else:
         url = IIQ_SCIM_USERS_EXT
         filter_list = []
-        if email is not None:
-            filter_list.append(''.join(('emails.value eq "', email, '"')))
-        if risk is not None:
-            filter_list.append(''.join(('urn:ietf:params:scim:schemas:sailpoint:1.0:User:riskScore ge ', str(risk))))
-        if active is not None:
-            filter_list.append(''.join(('active eq ', str(active).lower())))
+
+        # use custom filter
+        if filter is not None:
+            filter_list.append(filter)
+        else:
+            if email is not None:
+                filter_list.append(''.join(('emails.value eq "', email, '"')))
+            if active is not None:
+                filter_list.append(''.join(('active eq ', str(active).lower())))
+            # if risk is not None:
+            #     filter_list.append(''.join(('urn:ietf:params:scim:schemas:sailpoint:1.0:User:riskScore ge ', str(risk))))
+
         # Combine the filters
         if filter_list is not None and len(filter_list) > 0:
             filter_string = ' and '.join(filter_list)
             params = {'filter': filter_string}
+
+        LOG(f"VIPUL: params = {params}")
+        print(f"VIPUL: params = {params}")
```

Comment:
Please don't use LOG and print, you can use demisto.debug() and demisto.info() for debugging/logging.